### PR TITLE
#2595: Added a faint border around the current day to the design review calendar

### LINKED ISSUE #2595: DRC Debugs - Highlight the current day on the calendar

### PR DESCRIPTION
**Pull request by morrill-tw**
## Changes

Added a faint gray border around the current day to the design review calendar.

## Screenshots

Screenshot 2024-11-25 124941

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes #2595

### CODE REVIEW

**Comment by vsp05** [comment]
Great work! Could you remove your changes on ```yarn.lock```? Once you do that I'll approve!

**Comment by vsp05** [comment]
oh, also don't add a space in between # and 2595. If you remove the space, it will link this pr with the issue, which is what we want

File: yarn.lock [hunk 1]
```diff
@@ -6551,9 +6551,9 @@ __metadata:
   linkType: hard
 
 "caniuse-lite@npm:^1.0.0, caniuse-lite@npm:^1.0.30000981, caniuse-lite@npm:^1.0.30001109, caniuse-lite@npm:^1.0.30001125, caniuse-lite@npm:^1.0.30001587":
-  version: 1.0.30001629
-  resolution: "caniuse-lite@npm:1.0.30001629"
-  checksum: c5e646e1b309b2a70b01499e0f0fca3a54bc111212f121b32614fe925b8f24ff6c1832a4306ddadf35678fbb11a6a97f953be07ccdc96bce5b530a4f84f40c45
+  version: 1.0.30001680
```
Comment (alaneng-neu) [inline yarn.lock]: You shouldn't have any `yarn.lock` changes, because dependencies didn't change

**Review by alaneng-neu — CHANGES_REQUESTED** [review]

**Review by vsp05 — APPROVED** [review]
Great Work!

**Review by alaneng-neu — APPROVED** [review]
LGTM

**Review by walker-sean — APPROVED** [review]

**Pull request merged**